pages/classes/wise.md

```diff
@@ -26,10 +26,165 @@
+
+</header>
+
+You can cast the rituals below. A ritual takes 10 minutes to cast.
+
+<header>
+
+<section class="spells">
+
+<section class="spell">
+
+<header>
+
+#### Spirit Tongue
+
+<p class="subheading">1st-level Elementalist ritual</p>
+
+</header>
+
+You speak the language of the elements, such as stones, birds or even trees. What they have to say may or may not be helpful. You can ask the spirits for guidance, giving you an additional **d4** on one [Check](../../pages/rules/rolling/checks.md) appropriate for the kind of spirit you are speaking with.
+
+</section>
+
+<section class="spell">
+
+<header>
+
+#### Purify Elements
+
+<p class="subheading">1st-level Elementalist ritual</p>
+
+</header>
+
+You can sense and purify small amounts of [Close](../../pages/rules/distance.md) natural matter. Food and drink is rendered free of poison and disease, flames burn without smoke, and small animals are healed. Magical effects are immune to this ritual.
+
+</section>
+
+</section>
+
+### Elementalism
+
+<p class="subheading">1st-level Elementalist archetype feature</p>
+
+</header>
+
+You can cast spells. To cast a spell you must expend a spell slot. Each spell takes an [Action](../../pages/combat/actions.md) to cast. You regain your spell slots when you finish a [Rest](../../pages/rules/rests.md).
+
+Your **Spell Difficulty** is:
+
+<div class="example-roll">
+  8
+  +
+  <span class="bonus"> <span class="caption">Proficiency</span></span>
+  +
+  <span class="bonus"> <span class="caption">Wisdom</span></span>
+</div>
+
+<header>
+
+<section class="spells">
+
+<section class="spell">
+
+<header>
+
+#### Primordial Aid
+
+<p class="subheading">1st-level Elementalist spell</p>
+
+</header>
+
+A [Nearby](../../pages/rules/distance.md) creature regains **1d4 + Wisdom** [Stamina](../../pages/combat/stamina.md). When you cast this spell using a **Spell Slot** of 2nd level or higher, add an additional **1d4** for each slot level above 1st.
+
+</section>
+
+<section class="spell">
+
+<header>
+
+#### Primordial Aura
+
+<p class="subheading">1st-level Elementalist spell</p>
+
+</header>
+
+You channel a primordial spirit through up to three [Nearby](../../pages/rules/distance.md) creatures. For **1 minute** whenever they make an [Attack](../../pages/combat/attacks.md) or [Save](../../pages/rules/rolling/saves.md), they can add a **d4** to the roll. When you cast this spell using a **Spell Slot** of 2nd level or higher, you can target one additional creature for each slot level above 1st.
+
+</section>
+
+<section class="spell">
+
+<header>
+
+#### Primordial Strength
+
+<p class="subheading">1st-level Elementalist spell</p>
+
+</header>
+
+You channel a burst of raw, primordial energy through your body. You or a [Close](../../pages/rules/distance.md) creature regains **1d8 + Wisdom Stamina**. When you cast this spell using a **Spell Slot** of 2nd level or higher, add an additional **1d8** for each slot level above 1st.
```

Comment:
For a combat spell, Primordial Strength uses up an action and won't be used in combat